py_zipkin/transport/firehose_handler.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import socket
+
+import clog
+import staticconf
+
+
+ZIPKIN_CONFIG_NAMESPACE = 'zipkin'
+ZIPKIN_CONFIG_FILE = '/nail/srv/configs/zipkin.yaml'
+ZIPKIN_CONFIG_MIN_RELOAD_INTERVAL = 5
+
+ZIPKIN_FIREHOSE_IP = '169.254.255.254'
+ZIPKIN_FIREHOSE_PORT = 20182
+
+ZIPKIN_DEFAULT_FIREHOSE_ENABLED = False
+
+
+class FirehoseHandler(object):
```

Comment:
I'd call both this class and the file `udp_handler` or something like that, since it's not strictly firehose only.

Reply:
+1